src/data/clients/ror-v2-client.ts

```
@@ -1,6 +1,9 @@
 /**
  * Types for ROR API V2 responses and entities
  */
+
+import { LOCALHOST_URL } from 'src/data/constants'
```

ESLint

[error] 'LOCALHOST_URL' is defined but never used.

(no-unused-vars)